open-metadata-implementation/user-interfaces/presentation-server/cra-client/src/contexts/IdentificationContext.js

```diff
@@ -16,31 +17,28 @@ const IdentificationContextProvider = props => {
    * Get the home url for the browser
    */
   const getBrowserURL = pageName => {
-    if (process.env.NODE_ENV === "development") {
-      return "/" + "cocoView1" + "/" + pageName;
-    }
     return "/" + serverName + "/" + pageName;
   };
   /**
- * Get the url to use for rest calls 
- * @param {*} serviceName this is the viewservice name. 
- */
-const getRestURL = (serviceName) => {
-  const url =
-    window.location.protocol +
-    "//" +
-    window.location.hostname +
-    ":" +
-    window.location.port +
-    "/servers/" +
-    serverName +
-    "/" +
-    serviceName +
-    "/users/" +
-    userId;
-    console.log("rest url is ", url);
-  return url;
-};
+  * Get the url to use for rest calls 
+  * @param {*} serviceName this is the viewservice name. 
+  */
+  const getRestURL = (serviceName) => {
+    const url =
+      window.location.protocol +
+      "//" +
+      window.location.hostname +
+      ":" +
+      window.location.port +
+      "/servers/" +
+      serverName +
+      "/" +
+      serviceName +
+      "/users/" +
+      userId;
+      console.log("rest url is ", url);
```

Comment:
is this intended for developer debugging or the end user debugging (both)? This looks ok - but need to be careful not to introduce any personally identifiable info (that isnt already obvious from the browser env)

Reply:
@davidradl can you provide any input? I did not develop this piece.

Reply:
@planetf1 - this is part of the middleware that dos the multi-tenant support. I suggest we do not review the changes to the source code here unless there is a side effect .